reactive/openstack.py

```diff
@@ -251,7 +284,7 @@ def manage_loadbalancers_via_lb_consumers():
             lb_consumers.send_response(request)
             continue
 
-        remote_port = request.port_mapping.get(lb_port)
+        lb_port, remote_port = list(request.port_mapping.items())[0]
```

Comment:
so, after validation, we know there MUST be at least good pair...

```suggestion
        lb_port = next(iter(request.port_mapping))
        remote_port = request.port_mapping[lb_port])
```